plasmapy/particles/tests/test_ionization_state.py

```diff
@@ -52,6 +133,9 @@
 test_names = test_cases.keys()
 
 
+# TODO: Refactor these tests using fixtures
```

Comment:
~~There's not actually much of a reason to do this here, because the computational cost to initialize the `IonizationState` objects is low and there's no need to store instances between tests. You also have a separate set of parameters for each test, so you wouldn't reuse much~~ Oh, you mean the tests below, right? In that case, sure!

Reply:
- [ ] Create issue on using fixtures for these tests.  Be sure to include directional information.